src/main/com/diana/restaurant/entities/EntityPriorityForQueue.java

```diff
@@ -0,0 +1,37 @@
+package com.diana.restaurant.entities;
+
+public class EntityPriorityForQueue<E extends BaseEntity<Long>> {
```

Comment:
This should not be an entity, why would you want to do this?

Reply:
It is not needed

Reply:
solved